test/tasks/ods-build-python_test.go

```diff
@@ -0,0 +1,58 @@
+package tasks
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/opendevstack/pipeline/pkg/tasktesting"
+)
+
+func TestTaskODSBuildPython(t *testing.T) {
+	runTaskTestCases(t,
+		"ods-build-python-v0-1-0",
+		map[string]tasktesting.TestCase{
+			"task should build python flask app": {
+				WorkspaceDirMapping: map[string]string{"source": "python-flask-sample-app"},
+				PreRunFunc: func(t *testing.T, ctxt *tasktesting.TaskRunContext) {
+					wsDir := ctxt.Workspaces["source"]
+					ctxt.ODS = tasktesting.SetupGitRepo(t, ctxt.Namespace, wsDir)
+					ctxt.Params = map[string]string{
+						"python-image": "localhost:5000/ods/ods-python-toolset:latest",
+						"sonar-image":  "localhost:5000/ods/ods-sonar:latest",
```

Comment:
Image params should not be needed - in fact specifying those params should fail if the task does not consume them